<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filler</class>
 <widget class="QDialog" name="Filler">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory filler</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="2">
      <widget class="xHexSpin" name="leStartHex">
       <property name="inputMask">
        <string>HHHH</string>
       </property>
       <property name="text">
        <string>0000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="xHexSpin" name="leEndHex">
       <property name="inputMask">
        <string>HHHH</string>
       </property>
       <property name="text">
        <string>0000</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>End address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Start address</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>#</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>#</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pattern</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leBytes">
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
        </font>
       </property>
       <property name="inputMask">
        <string>HH:hh:hh:hh:hh:hh:hh:hh</string>
       </property>
       <property name="text">
        <string>00:::::::</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mask</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leMask">
       <property name="font">
        <font>
         <family>DejaVu Sans Mono</family>
        </font>
       </property>
       <property name="inputMask">
        <string>HH:hh:hh:hh:hh:hh:hh:hh</string>
       </property>
       <property name="text">
        <string>FF:::::::</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="cbMethod">
       <item>
        <property name="text">
         <string>Mask</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Put</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Or</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>And</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Xor</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbFill">
       <property name="text">
        <string>Fill</string>
       </property>
       <property name="icon">
        <iconset resource="../xpeccy.qrc">
         <normaloff>:/images/fill.png</normaloff>:/images/fill.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xHexSpin</class>
   <extends>QLineEdit</extends>
   <header location="global">xgui/xgui.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../xpeccy.qrc"/>
 </resources>
 <connections/>
</ui>
